### PR TITLE
TNDO-2646 Allow configuring src and dist path for upload

### PR DESCRIPTION
**Pull request by superquadratic**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -26,45 +26,62 @@ program
...
 
     const host = options.host || 'store.leanix.net';
     const tokenhost = options.tokenhost || 'app.leanix.net';
     const msg = `Bundling and uploading your project to the LeanIX Store (${host})...`;
+    const url = `https://${host}/services/torg/v1/assetversions/${id}/payload`;
+
+    const builder = new Builder(console);
+    const uploader = new Uploader();
+
     console.log(chalk.yellow(chalk.italic(msg)));
```
Comment (aleics-lx) [inline src/app.ts:81]: I guess we could use the `msg` inline as well here, to make it consistent.

File: src/file.helpers.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ export function loadCliConfig(packageJson = loadPackageJson()): CliConfig {
   const leanixReportingCli = packageJson.leanixReportingCli || {};
 
   return {
+    srcPath: leanixReportingCli.srcPath ?? './src',
```
Comment (aleics-lx) [inline src/file.helpers.ts]: I guess `'./src'` is the same as `getProjectDirectoryPath('src')`, which was the one previously used, right?

File: src/bundler.spec.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import * as tar from 'tar';
+import * as fs from 'fs';
+import * as asyncHelpers from './async.helpers';
+import { Bundler } from './bundler';
+import * as fileHelpers from './file.helpers';
+
+describe('Bundler', () => {
+  const writeFileAsync = jest.spyOn(asyncHelpers, 'writeFileAsync').mockResolvedValue(undefined);
+  const tarC = jest.spyOn(tar, 'c').mockResolvedValue(undefined as never);
```
Comment (aleics-lx) [inline src/bundler.spec.ts:9]: So the value should be `never` `undefined`? 🤔 😜 🥁

File: src/bundler.ts [hunk 4]
```diff
@@ -30,11 +29,6 @@ export class Bundler {
     return writeFileAsync(metadataFile, JSON.stringify(metadata));
   }
 
-  private createTarFromSrcFolderAndAddToDist(srcPath: string, distPath: string) {
-    const files = fs.readdirSync(srcPath);
-    return tar.c({ gzip: true, cwd: srcPath, file: join(distPath, 'src.tgz') }, files);
```
Comment (aleics-lx) [inline src/bundler.ts]: If I understand this right, `src.tgz` is not used anyway for the store or when uploading a report, right? Or is it not even uploaded?

**Review by aleics-lx — APPROVED** [review]
LGTM! ✅ 

I have a few minor comments, but to mainly understand specific changes 🚀

File: src/app.ts (hunk 1 above)
Comment (superquadratic) [reply src/app.ts]: Good catch, looks cleaner now 😃

File: src/file.helpers.ts (hunk 2 above)
Comment (superquadratic) [reply src/file.helpers.ts]: Yes, `getProjectDirectoryPath()` is just the current working directory.

File: src/bundler.spec.ts (hunk 3 above)
Comment (superquadratic) [reply src/bundler.spec.ts:9]: Yeah, this is really weird. Mock typings for Jest apparently always take the last overload of function, which in this case has a return type of `never`... 🤷‍♂️

File: src/bundler.ts (hunk 4 above)
Comment (superquadratic) [reply src/bundler.ts]: Yes, I talked to @psurrey, @blandsbergleanix and @nikolasrist, and nobody is aware of any usage of it. It's also not used in Pathfinder and I double-checked that reports can be uploaded without any issue without a `src.tgz` in the bundle.

**Review by aleics-lx — APPROVED** [review]
👏

**Pull request merged**